hr.ibicom.treemly.yml

```diff
@@ -0,0 +1,27 @@
+app-id: hr.ibicom.treemly
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+command: treemly
+finish-args:
+  - --socket=fallback-x11
+  - --share=network
+  - --share=ipc
+  - --socket=wayland  
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+modules:
+  - name: treemly
+    buildsystem: simple
+    only-arches:
+      - x86_64    
```

Comment:
you need to add a flathub.json https://docs.flathub.org/docs/for-app-authors/maintenance#flathubjson